vmdb/lib/token_manager.rb

```diff
@@ -28,7 +22,13 @@ def self.class_initialize(name = DEFAULT_NS, options = {})
   end
 
   def self.global_token_store(namespace)
-    @global_token_store[namespace] ||= {}
+    @token_caches[namespace] ||= begin
+      memcache_server = VMDB::Config.new("vmdb").config[:session][:memcache_server] || "127.0.0.1:11221"
+      Dalli::Client.new(memcache_server,
+                        :namespace  => "MIQ:VMDB:TOKENS:#{namespace.upcase}",
```

Comment:
Oh nvm...I see this is a generic Memcache manager...what confused me is that the class is called TokenManager, which in my mind auto-tied it to rest api.